journal/original/17-chapter-14.adoc

```diff
@@ -1225,6 +1250,9 @@ or of inward fascinations and snares of the enemy.]
 Wherefore feeling the judgment of truth rise against it,
 I gave forth the following as a warning to all that were concerned therein
 
+[.embedded-content-document.letter]
```

Comment:
This is tricky, but I'm pretty sure we should label it an `epistle` since it's directed to friends in general, warning about an apostate friend named John Perot.